config/environments/production.rb

```diff
@@ -49,7 +49,7 @@
   config.log_tags = [:request_id]
 
   # Use a different cache store in production.
-  config.cache_store = :file_store
+  config.cache_store = :redis_cache_store
```

Comment:
will this pick the redis connection info from the redis initialiser?
or should we include redis connection options like
`:redis_cache_store, { url: '' }`

Reply:
Would be good to give it an explicit url, is there an env_var we can use?

Reply:
no, it only available under VCAP_SERVICES at the moment.
https://github.com/DFE-Digital/register-trainee-teachers/blob/master/config/initializers/redis.rb#L3
maybe the code  can be placed in a lib class and referenced from there?

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/1001/commits/90406c1f9166f3b7e96f61fb9b8e101a6cf7e7c1